packages/babel-core/src/config/caching.ts

```diff
@@ -16,56 +15,62 @@ export type SimpleCacheConfigurator = SimpleCacheConfiguratorFn &
   SimpleCacheConfiguratorObj;
 
 type SimpleCacheConfiguratorFn = {
-  (boolean): void,
-  <T>(handler: () => T): T,
+  (a: boolean): void;
```

Comment:
I'm going to push a commit to give a name to all these auto-generated parameter names.
I know that it's impossible to do it with an automatic converter, but I really dislike the `a`s everywhere :sweat_smile: